profile/README.md

```diff
@@ -1,42 +1,12 @@
 ## Welcome to the Autoware Foundation! 👋
 
-![Autoware_RViz](https://user-images.githubusercontent.com/63835446/158918717-58d6deaf-93fb-47f9-891d-e242b02cba7b.png)
+![awf_logo_white_background](https://user-images.githubusercontent.com/63835446/162110040-7cbceb06-5854-4af4-98ff-2ab77629bacc.png)
 
-The [Autoware Foundation](https://www.autoware.org/) manages the development of Autoware, the world's leading open-source software project for autonomous driving, and sponsors Autoware-based projects all over the world. Autoware was started in 2015 and is being used by over 100 companies on more than 30 vehicles in over 20 different countries worldwide.
-
-[![Discord](https://img.shields.io/discord/953808765935816715?label=Join%20Autoware%20Discord&style=for-the-badge)](https://discord.gg/Q94UsPvReQ)
```

Comment:
Could you leave this in the org profile too?

Reply:
Just to clarify: Only Discord? or including the AWF description?

Reply:
Only Discord.

Reply:
Added the badge back in as requested